electrum/util.py

```diff
@@ -792,10 +792,10 @@ def format_satoshis(
 
 FEERATE_PRECISION = 1  # num fractional decimal places for sat/byte fee rates
 _feerate_quanta = Decimal(10) ** (-FEERATE_PRECISION)
-UI_UNIT_NAME_FEERATE_SAT_PER_VBYTE = "sat/byte"
-UI_UNIT_NAME_FEERATE_SAT_PER_VB = "sat/b"
-UI_UNIT_NAME_TXSIZE_VBYTES = "bytes"
-UI_UNIT_NAME_MEMPOOL_MB = "MB"
+UI_UNIT_NAME_FEERATE_SAT_PER_VBYTE = "sat/vbyte"
+UI_UNIT_NAME_FEERATE_SAT_PER_VB = "sat/vB"
+UI_UNIT_NAME_TXSIZE_VBYTES = "vbytes"
+UI_UNIT_NAME_MEMPOOL_MB = "MvB"
```

Comment:
I am a bit conflicted about this last one ("MvB")...

Reply:
About block 828682,
- mempool.space says
  ```
  Size	1.54 MB
  Weight	3.99 MWU
  ```
- blockstream.info says
  ```
  Size	1543.694 KB
  Virtual size	999 vKB      <----
  Weight units	3993.305 KWU
  ```

For the mempool diagram,
- johoe's site uses vMB:
  <details>
   <summary>pic</summary>
  
  ![johoe](https://github.com/spesmilo/electrum/assets/29142493/4fff81c9-2c41-4f6b-9c04-859c759d40cc)
  </details>

- mempool.space uses MvB:
  <details>
   <summary>pic</summary>
  
  ![mempoolspace](https://github.com/spesmilo/electrum/assets/29142493/791292c5-95f8-4b5c-b25a-556574cd63ce)
  </details>
  
-----

Thoughts?

Reply:
Is it a unit that is virtual, or is it a virtual unit of itself? :)

Reply:
no strong opinion here, but 'virtual megabyte' feels more appropriate than 'mega virtualbyte'

Reply:
All right, let's use `vMB` then. I think it looks better, and hopefully less people get confused by it.